addons/mail/static/src/components/discuss/tests/discuss_inbox_tests.js

```diff
@@ -411,6 +411,59 @@ QUnit.test('"reply to" composer should send message if message replied to is not
     assert.verifySteps(['message_post']);
 });
 
+QUnit.test('error notifications should not be shown in inbox', async function (assert) {
+    assert.expect(3);
+
+    Object.assign(this.data.initMessaging, {
+        needaction_inbox_counter: 1,
+        channel_slots: {
+            channel_channel: [{
+                channel_type: "channel",
+                id: 20,
+                is_pinned: true,
+                name: "General",
+            }],
+        },
+    });
+    this.data['res.partner'].records = [{
+        email: "testpartnert@odoo.com",
+        id: 11,
+        name: "TestPartner",
+    }];
```

Comment:
I don't think this part is necessary, that was for mentions in another test I believe.